composer.json

```diff
@@ -14,10 +14,14 @@
         "php": ">=5.5"
     },
     "require-dev": {
-        "phpunit/phpunit": ">4.8",
-        "phpdocumentor/phpdocumentor": ">=2.9",
+        "jakoch/phantomjs-installer": "^2.1",
+        "phpunit/phpunit": "=4.8",
```

Comment:
I think the phpunit/phpunit dependency should be `">4.8"`, or at least `"^4.8"`. There's no reason to force it down to 4.8.0.

I have this test suite running on `5.6.5` without issues.